lib/widgets-page.php

```diff
@@ -37,11 +37,11 @@ function gutenberg_widgets_init( $hook ) {
 		);
 		return;
 	}
-	if ( ! in_array( $hook, array( 'appearance_page_gutenberg-widgets' ), true ) ) {
+	if ( ! in_array( $hook, array( 'appearance_page_gutenberg-widgets', 'gutenberg_customizer' ), true ) ) {
```

Comment:
This enqueues everything in `admin_enqueue_scripts`. I'm not sure we should be using that. From memory the Customizer has a dedicated hook which is where you're supposed to enqueue assets.